spec/features/notes_on_merge_requests_spec.rb

```diff
@@ -94,8 +94,8 @@
           end
 
           within("#note_#{note.id}") do
-            should have_css(".note_edited_ago")
-            find(".note_edited_ago").text.should match(/less than a minute ago/)
+            is_expected.to have_css(".note_edited_ago")
+            expect(find(".note_edited_ago").text).to match(/less than a minute ago/)
```

Comment:
Line is too long. [84/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.